pkg/acme/x/acme/types.go

```diff
@@ -220,7 +220,7 @@ type Order struct {
 	// StatusProcessing, StatusValid, and StatusInvalid.
 	Status string
 
-	// Expires is the teimstamp after which the server will consider the order invalid.
+	// Expires is the timestamp after which the server will consider the order invalid.
```

Comment:
package pkg/acme/x is a verbatim copy of an external package, so I'd suggest to ignore it.